packages/block-editor/src/components/url-popover/image-url-input-ui.js

```diff
@@ -0,0 +1,312 @@
+/**
```

Comment:
Not a blocker for this PR, but I think we should create a readme and add some test cases for this component.

Reply:
I think this will be deprecated by the new LinkEditor.